sft-n-hrd.js

```diff
@@ -1,19 +1,32 @@
-const inchesToCmsConvertionFactor = 2.54
+
+// Event listener para las cajas de input
+const enterButton= document.getElementById("enterbutton");
+
+enterButton.addEventListener('click', function(){
+  let softLenght = document.getElementById("softbox").value; //Getting info from softbox
+  let hardLenght = document.getElementById("hardbox").value; //Getting info from hardbox
+  console.log(softLenght, hardLenght);
+  document.getElementById("result").innerHTML = (hardLenght - softLenght); //Printing the difference on the screen
+});
+
+/*
+
+//Conversion de pulgadas a centimetros 
```

Comment:
Can we delete all this old code we're not using for now?

Reply:
> Can we delete all this old code we're not using for now?

No, sorry